packages/magento-product/components/ProductPageMeta/ProductPageMeta.tsx

```diff
@@ -16,24 +18,20 @@ export function ProductPageMeta(props: ProductPageMetaFragment) {
     __typename,
   } = props
   const productLink = useProductLink({ url_key, __typename })
-  const canonical = useCanonical(productLink)
 
   return (
     <PageMeta
       title={meta_title ?? name ?? ''}
       metaDescription={meta_description ?? name ?? ''}
       canonical={productLink}
+      openGraphImage={media_gallery?.[0]?.url}
     >
       {sku && (
         <>
           <meta property='type' content='product' key='og-type' />
```

Comment:
We're using a type here, but later we're using og:type. Is that something we need to move to PageMeta as well?